spk/syncthing/src/service-setup.sh

```diff
@@ -0,0 +1,29 @@
+# Setup environment
+PATH="${SYNOPKG_PKGDEST}/bin:${PATH}"
+SYNCTHING="${SYNOPKG_PKGDEST}/bin/syncthing"
+CONFIG_DIR="${SYNOPKG_PKGDEST}/var"
+SYNCTHING_OPTIONS="-home=${CONFIG_DIR}"
+
+# Read additional startup options from /usr/local/syncthing/var/options.conf
+if [ -f ${CONFIG_DIR}/options.conf ]; then
+    source ${CONFIG_DIR}/options.conf
+fi
+
+# Add special options to start-stop-daemon command
+SERVICE_OPTIONS = SYNCTHING_OPTIONS
```

Comment:
This should not work. I propose to rename `SYNCTHING_OPTIONS` into `SERVICE_OPTIONS`

Reply:
Indeed it doesn't work, in my test-branch I already updated that but just didn't push it yet.
But the name needs to stay `SYNCTHING_OPTIONS` because users could have setup some custom options during previous versions via `${CONFIG_DIR}/options.conf`. 
That `options.conf` is kept during the upgrade, so for backwards-compatibility it needs this.

```sh
#!/bin/sh
# This file will be added to start-stop-status script
# For possible options see syncthing --help

# Example: uncomment this to start syncthing with all devices paused
# SYNCTHING_OPTIONS="${SYNCTHING_OPTIONS} -paused"
```

Reply:
OK. Have not read inside `options.conf`. So I wait for your review request.